examples/plot_classification_example.py

```diff
@@ -0,0 +1,258 @@
+"""
+Measuring Individual and Group Variable Importance for Classification
+======================================================================
+In this example, we show on the Iris dataset how to measure variable importance for
+classification tasks. We use two different variable importance methods: CPI and
+PermutationImportance with two different classifiers: LogisticRegressionCV and
```

Comment:
In the explanation use the common name of the model and not the name of the function.
Can you replace `LogisticRegressionCV` by `Logistic Regression` and `LinearSVC` by `Linear Support Vector Classification`?

Reply:
Same remark for `Permutation Importance` use `PFI`.

Reply:
done